docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,108 @@
+
+After considering needs, chosen IAM solution and taking into consideration the security requirements, the last step is to test it out thoroughly before deploying it to a production environment.
+
+## Internal vs external users
+
+As a first step, determine where your Grafana users are located. Are the users located within your organization, or are they outside your organization?
+
+If the users are within your organization, Grafana might be able to integrate with those users through an identify provider.
+
+If the users are outside your organization, you must provide anonymous access to Grafana, which is not enabled by default.
+
+For information about enabling anonymous access, refer to the [documentation](../../setup-grafana/configure-security/configure-authentication/_index.md#anonymous-authentication)
+
+## Ways to organize users
+
+1. **Security**: Different groups of users or customers should only have access to their intended resources.
+1. **Simplicity**: Reduce scope of dashboards and resources available.
+1. **Cost attribution**: Track and bill costs to their customers, departments, or divisions.
+1. **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana Teams
+
+It makes sense to organize users in [Teams] in order to assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team. A user can belong to multiple teams.
+
+A user can be a Member or an Administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team Administrators can add members to a team and update its settings, such as the team name, team member’s team roles, UI preferences, and the default dashboard to be displayed upon login for the team members.
+
+Teams is a perfect solution to work with a small subset of users. Teams can share resources among other teams.
+
+### Users in Grafana Organizations
+
+[Grafana Organizations] was born as a concept to isolate users from dashboards and datasources by having multiple organizations under a single instance of Grafana. This means that users under different organizations won't share any resources such as dashboards, folders, and datasources.
+
+Grafana Organizations provide a measure of isolation within Grafana by default. The intention is to present different user experiences, which give the sense of different instances of Grafana within a single instance. However, **we recommend aginast Grafana Organizations** because they lack the scalability of [Folders].
```

Comment:
```suggestion
By default, organizations provide a measure of isolation within Grafana. They can be used to present different user experiences, which gives the appearance that there are multiple instances of Grafana within a single instance. However, because they lack the scalability of [Folders], we do not recommend using organizations as a way to group users.
```

Reply:
Changed